<launch>
	<arg name="speed_topic" value="set_speed" />
	<arg name="steer_topic" value="set_angle" />

	<node name="gear_control" pkg="gear_control" type="gear_control_node" output="screen">
		<param name="speed_topic" value="$(arg speed_topic)"/>
		<param name="steer_topic" value="$(arg steer_topic)"/>
		<param name="pwm_pca9685" value="100"/>
	</node>

	<node name="gear_debug" pkg="gear_control" type="gear_debug" output="screen">
		<param name="speed_topic" value="$(arg speed_topic)"/>
		<param name="steer_topic" value="$(arg steer_topic)"/>
	</node>

	<node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" args="gear_debug"/>

</launch>
